controller/src/main/java/io/pravega/controller/task/TaskSweeper.java

```diff
@@ -76,9 +76,9 @@ public boolean isReady() {
     public CompletableFuture<Void> sweepFailedProcesses(final Supplier<Set<String>> runningProcesses) {
         return withRetriesAsync(taskMetadataStore::getHosts, RETRYABLE_PREDICATE, Integer.MAX_VALUE, executor)
                 .thenComposeAsync(registeredHosts -> {
-                    log.info("Hosts {} have ongoing tasks", registeredHosts);
+                    log.debug("Hosts {} have ongoing tasks", registeredHosts);
                     registeredHosts.removeAll(withRetries(runningProcesses, UNCONDITIONAL_PREDICATE, Integer.MAX_VALUE));
-                    log.info("Failed hosts {} have orphaned tasks", registeredHosts);
+                    log.debug("Failed hosts {} have orphaned tasks", registeredHosts);
```

Comment:
we should keep these at info.

Reply:
Ok. yes, since it helps explain what hosts/tasks were part of the failover.

Reply:
fixed